proposals/0173-Read-Generic-Network-Signal-data.md

```diff
@@ -56,7 +56,7 @@ The proposed Policy Table structure assumes that SDL enums are defined in the RP
 #### Schema file location and updates
 A base JSON schema file for SDL core and proxy will be kept locally and an additional one in the OEM cloud. OEMs can update the schema file with OEM specific items, which SDL core would be able to download over the cloud along with policy updates; SDL proxy would still process the local(static) copy of schema for existing data items. For OEM specific additional items, the OEM app would process a generic _Object_ to retrieve vehicle data item response _struct_. This new design would ensure flexibility to add OEM specific vehicle data items while maintaining access control via policies the same way it is done currently.
 
-* OEM's schema file should always be baselined with latest version of standardized schema file from open SDL. OEM can then further add OEM specific custom vehicle data items.
+* OEM's schema file should always be baselined with latest version of standardized schema file from open SDL. OEM can then further add OEM specific custom vehicle data items. This ensures that any version of core which implement this proposal would be able to process the latest version of standardized vehicle data items schema, hence would be able to support all future additions for new standardized vehicle data items.
```

Comment:
Please update: 

This ensures that any version of core which **implements** this proposal would be able to process the latest version of standardized vehicle data **item** schema, **and** hence would be able to support all future additions for new standardized vehicle data items.

Reply:
Done